docs/assets/js/charts.js

```diff
@@ -382,14 +402,51 @@ function createHistoryChart(canvas, actionBar)
                 actionBar.prepend(buttons);
             }
 
+            const tooltipTitleCallback =
+                function(tooltipItem, data)
+                {
+                    // In the history chart, tooltips are rendered for individual data points.
+                    // Hence, the tooltipItem array contains exactly one element
+                    const dataPoint = data.datasets[tooltipItem[0].datasetIndex].data[tooltipItem[0].index];
+
+                    const date = dataPoint.x;
+                    const dateRange = dataPoint.dateRange;
+
+                    const suffix = (dataPoint.infoText === undefined)
+                        ? ''
+                        : ' (' + dataPoint.infoText + ')';
```

Comment:
Nit: Wouldn't is be more straight forward to set `row['_infoText'] = ''`? Then you don't need this condition.

Reply:
I thought I’d rather decouple the presentation (putting the info text in parentheses after the title) from the data itself. This keeps the data array rather clean I think even if this conditional is the consequence.